ckanext/canada/templates/public/base.html

```diff
@@ -31,7 +31,7 @@
 {%- endblock -%}
 
 {% block adobe_analytics %}
-  <script src="{{ h.adobe_analytics_js() }}"></script>
+  <script src="{{ h.adobe_analytics_id() }}"></script>
```

Comment:
How about
```jinja2
{% if h.adobe_analytics_js() %}
  <script src="{{ h.adobe_analytics_js() }}"></script>
{% endif %}
```
so that we don't end up with an empty tag when this isn't configured.

Also add the new config option to the readme so new users know it exists

Reply:
Yes that's better.